content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/sboms-and-attestations.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "SBOMs and Attestations in Chainguard Enforce"
+type: "article"
+description: "An overview of how Chainguard Enforce handles attestations and SBOMs"
+date: 2023-03-19T15:56:52-07:00
+lastmod: 2023-03-19T15:56:52-07:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-enforce-kubernetes"
+weight: 600
+toc: true
+---
+
+In order to fully leverage the capabilities offered by Chainguard Enforce, it's important to establish a process for creating quality *Software Bills of Materials* (SBOMs). An [SBOM](/open-source/sbom/) is a formal record that contains the details and supply chain relationships (such as dependencies) of the components used in building software. Chainguard Enforce can capture SBOM data and use it to help you find images and packages in your ecosystem, as well as apply and enforce security policies to your runtime environments. To get the best results from these features, SBOMs should be attached to the images in your registry as signed attestations. 
+
+[Cosign](/open-source/sigstore/cosign/an-introduction-to-cosign/)  — a part of the Sigstore project — supports software artifact signing, verification, and storage in an [OCI (Open Container Initiative)](/open-source/oci/) registry. The `cosign` command line tool offers two subcommands that you can use to associate an SBOM with a container image and then upload them to a registry: `cosign attach` and `cosign attest`. 
+
+However, these commands don't work the same way, and Chainguard Enforce treats SBOMs that have been attached differently from those that have been attested. This guide outlines the differences between these two subcommands and provides guidance for when you might want to use one over the other.
+
+
+## Comparison of attaching and attesting
+
+The following table presents a high-level comparison of the `cosign attach` and `cosign attest` commands.
+
+|   | `cosign attach` | `cosign attest` |
```

Comment:
@tcnghia Could you review this entire table for accuracy? Adam seemed to think that how we ingest .sbom and .att may have changed recently